bindings/ergo-lib-wasm/src/ast.rs

```diff
@@ -123,6 +171,19 @@ impl Constant {
             .collect())
     }
 
+    /// Extract `Coll[Coll[Byte]]` as byte array
+    pub fn to_coll_coll_byte_array(&self) -> Result<Vec<Uint8Array>, JsValue> {
```

Comment:
```suggestion
    pub fn to_coll_coll_byte(&self) -> Result<Vec<Uint8Array>, JsValue> {
```
`array` is redundant since `coll_coll` already conveys the type.